docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -4820,24 +4820,40 @@ client.resolvers.invalidateMentionSuggestions();
 
 ## Utilities
 
-### getMentionedIdsFromCommentBody [#get-mentioned-ids-from-comment-body]
```

Comment:
I always struggle to decide what should be kept as "deprecated" in the docs for posterity vs what can be removed/replaced, `getMentionedIdsFromCommentBody` is a very niche utility so I'm removing it.